dist/pom.xml

```diff
@@ -857,6 +857,26 @@
             <dep>io.camunda.optimize:optimize-webjar</dep>
           </ignoredUnusedDeclaredDependencies>
         </configuration>
+        <executions>
+          <execution>
+            <id>unpack</id>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <phase>generate-resources</phase>
```

Comment:
:question: Any reason for using the `generate-resources` phase instead of `prepare-package`? I think `prepare-package` seems more fitting because the native library is only needed in the dist build.